Makefile

```diff
@@ -104,6 +106,9 @@ $(CONVERSION_GEN): $(TOOLS_DIR)/go.mod
 $(CERTMANAGER_UPDATE): $(TOOLS_DIR)/go.mod
 	cd $(TOOLS_DIR); go build -tags=tools -o $(BIN_DIR)/certmanager-update ./certmanager
 
+$(RELEASE_NOTES) : $(TOOLS_DIR)/go.mod
+	cd $(TOOLS_DIR) && go build -o $(RELEASE_NOTES_BIN) -tags tools sigs.k8s.io/cluster-api/hack/tools/release
```

Comment:
Not a required change, but instead of fully spelling out `sigs.k8s.io/cluster-api/hack/tools/release`, we could use `./release` instead, couldn't we?